pkg/tests/BUILD

```diff
@@ -536,3 +536,42 @@ filegroup(
     ],
     visibility = ["//visibility:public"],
 )
+
```

Comment:
This file needs some visible division to make it easier to browse.  Perhaps a subject of a future change.